### ISSUE TITLE
Directly call logits node in Keras models

### ISSUE THREAD

**Issue by wielandbrendel** [bethgelab/foolbox#65]
Cleverhans directly searches for the logits output by tracking the node before the softmax:

https://github.com/tensorflow/cleverhans/blob/master/cleverhans/utils_keras.py

I am not sure whether the same trick works regardless of the backend (doubt it) but it would be interesting to take a look.

**Comment by jonasrauber** [comment]
closed by #69
